ogre2/src/Ogre2Camera.cc

```diff
@@ -24,11 +24,17 @@
 #include "ignition/rendering/ogre2/Ogre2SelectionBuffer.hh"
 #include "ignition/rendering/Utils.hh"
 
+/// \brief Private data for the Ogre2Camera class
+class ignition::rendering::Ogre2CameraPrivate
+{
+};
+
 using namespace ignition;
 using namespace rendering;
 
 //////////////////////////////////////////////////
 Ogre2Camera::Ogre2Camera()
+  : dataPtr(new Ogre2CameraPrivate)
```

Comment:
Consider using `make_unique` for all `dataPtr`s

Reply:
changed to `make_unique` d2b2979df0ced5ac848da30d8ecb021084c7ea53